### ISSUE TITLE
Add EARL as optional export format for Axe DevTools 

### ISSUE THREAD

**Issue by mgifford** [dequelabs/axe-core#3413]
axe-earl is output should be accessible from Axe DevTools. The existing JSON format should be more easily accessible, but I should be able to bring the results over to something like WCAG-EM easily.

**Comment by 970Design** [comment]
This would be a great feature, as still over a year later there isn't an easy way to get EARL results from axe.

**Comment by mgifford** [comment]
@970Design I still like this idea.

This is likely something that would need to be an add-on or at least a pull-request to get in. Not sure it makes any sense to expect Deque to do this.

Not that it wouldn't be a great way to support a W3C standard for reporting.
